tests/src/main/java/com/hazelcast/stabilizer/tests/synthetic/SyntheticTest.java

```diff
@@ -172,24 +176,24 @@ public void doRun() throws Exception {
 
                 int partitionId = nextPartitionId();
 
-                ICompletableFuture f = invoke(partitionId);
+                ICompletableFuture<Object> future = invoke(partitionId);
```

Comment:
It removes the "unchecked" warning on the future usage. Since this is a default warning in IntelliJ it removes that noise for most of the people using it.